apps/api/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "api",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
```

Comment:
Why is this unlicensed? It should be MIT. Did you add this value manually?

Reply:
No, it is a default value only, I didn't make any changes in this